.circleci/config.yml

```diff
@@ -3,23 +3,19 @@ version: 2.1
 orbs:
   win: circleci/windows@2.2.0
 
+# All windows builds use the executor win/default and modify the Python version through
+# the conda environment.
 executors:
-  py36:
-    docker:
-      - image: 350138855857.dkr.ecr.eu-west-2.amazonaws.com/kedro-builder:3.6
-    resource_class: medium+
-  py37:
-    docker:
-      - image: 350138855857.dkr.ecr.eu-west-2.amazonaws.com/kedro-builder:3.7
-    resource_class: medium+
-  py38:
+  docker:
+    parameters:
+      python_version:
+        type: string
     docker:
-      - image: 350138855857.dkr.ecr.eu-west-2.amazonaws.com/kedro-builder:3.8
+      - image: 350138855857.dkr.ecr.eu-west-2.amazonaws.com/kedro-builder:<<parameters.python_version>>
     resource_class: medium+
 
 commands:
   setup_conda:
-    description: Activate conda environment
```

Comment:
These descriptions weren't really consistent throughout the file and don't appear anywhere on CircleCI (the human readable `name` does instead, which everything has already) and the names of all the commands are now pretty self-explanatory so I removed them.